compiler/commands.nim

```diff
@@ -141,6 +149,10 @@ proc expectArg(switch, arg: string, pass: TCmdLinePass, info: TLineInfo) =
 proc expectNoArg(switch, arg: string, pass: TCmdLinePass, info: TLineInfo) =
   if arg != "": localError(info, errCmdLineNoArgExpected, addPrefix(switch))
 
+var
```

Comment:
Because `gNotes` gets overridden changing verbosity level, see `of "verbosity"` case below. This way we keep track what were `hintName:on` in `enableNotes` and `hintName:off` in `disableNotes`.

Once we remove this verbosity completely (some next PR) then this will be no longer necessary.